shovel-ts/src/index.ts

```diff
@@ -0,0 +1,78 @@
+type hex = `0x${string}`;
```

Comment:
type are typically pascal case
```suggestion
type Hex = `0x${string}`;
```